pkg/controllers/clusterrole/clusterrole_controller.go

```diff
@@ -44,20 +50,76 @@ func newReconciler(mgr manager.Manager, kubeClient kubernetes.Interface) reconci
 	}
 }
 
+var _ handler.EventHandler = &enqueueRequestForClusterRole{}
+
+type enqueueRequestForClusterRole struct{}
+
+func (e *enqueueRequestForClusterRole) enqueue(clusterroleName string, q workqueue.TypedRateLimitingInterface[reconcile.Request]) {
+	clusterName, err := getClusterNameFromClusterRoleName(clusterroleName)
+	if err != nil {
+		klog.Info("error getting cluster name from clusterrole name", err)
```

Comment:
I do not think the code can run here, we already validated the clusterrole by `isValiedClusterRole`, maybe just delete this error print line, if you still want to keep it, please change Info to Error, and print out the clusterrole name.

Reply:
Agree, print deleted.